crates/sui-benchmark/src/workloads/workload_configuration.rs

```diff
@@ -31,22 +28,43 @@ impl WorkloadConfiguration {
                 batch_payment,
                 batch_payment_size,
                 shared_counter_hotness_factor,
+                mut weights,
+                mut workload_parameters,
                 ..
             } => {
+                // TODO: temporary code for backward compatibility. Remove it
```

Comment:
I think we should remove this backward compatibility code. Thinking more about it, it seems like fixing sui-ops with new parameters is the way to go (apologies i know i asked for it)